### DISCUSSION TITLE
Unable to Load Cal File

### DISCUSSION THREAD

**Discussion by Tino-Seibold** [jankae/LibreVNA#274] [Q&A]
Hello, 

the SCPI command :VNA:CAL:LOAD? only works for me if I enter the file name without a path. As soon as I enter the path to the same file, I get a “False” return. 

How can I fix this? I would like to avoid a fixed name in the code, because I would like to select a file with a selection window, which can also be saved in another folder

**Comment by jankae** [comment]
Hm, works for me:

> VNA:CAL:LOAD? /home/jan/Projekte/LibreVNA/test.cal
TRUE

Can you please post the exact command you send? Maybe there are problems with specific file names/paths.

**Reply by Tino-Seibold** [reply]
In my case, the following call works:
cal_loaded=vna_device.query(“:VNA:CAL:LOAD? TEST.cal”)

and with this I get the return value “false”:
cal_loaded=vna_device.query(“:VNA:CAL:LOAD? C:\Documents\SCPI_Examples\TEST.cal”)

**Reply by jankae** [reply]
Very strange. Switching to windows doesn't immediately help in reproducing the problem. I tried this over putty:

> :VNA:CAL:LOAD? D:\jan\Documents\Privat\LibreVNA\test.cal
TRUE

Then I switched to a python script:
```python
#!/usr/bin/env python3
from libreVNA import libreVNA
vna = libreVNA('localhost', 19542)
result = vna.query(":VNA:CAL:LOAD? D:\jan\Documents\Privat\LibreVNA\test.cal")
print(result)
```
...and:

> FALSE

The exact same command fails. But why? The GUI actually has a hint here. It shows an error message:
![image](https://github.com/user-attachments/assets/cae73750-fe60-4120-ab22-2a30f20404d5)
Python is converting the `\t` in `\test.cal` to a tab and that will obviously not work. Turning the backslashes into `/` helps here.

But you have `\T` instead of `\t` in your example. That is not converted to a tab. In fact, if I rename my file to use a capital T and adjust the python script accordingly:
```python
#!/usr/bin/env python3
from libreVNA import libreVNA
vna = libreVNA('localhost', 19542)
result = vna.query(":VNA:CAL:LOAD? D:\jan\Documents\Privat\LibreVNA\Test.cal")
print(result)
```
everything works:

> TRUE

So I still have no idea why you have a problem here. The only ideas I have are:

- make sure the file is really at that location (I know, you probably triple-checked already)
- check the GUI for error messages
- maybe try it with `/` instead of `\`

**Reply by Tino-Seibold** [reply]
thanks for the answer, the problem was elsewhere. It was the location that was wrong but that is now fixed and the program works
